### PR TITLE
[Tests] update test tools

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: tests/exchanges/__init__.py [hunk 1]
```diff
@@ -49,6 +51,17 @@ class MockedCCXTConnector(CCXTConnector):
     def get_name(cls):
         return DEFAULT_EXCHANGE_NAME
 
+    async def load_symbol_markets(self, reload=False, forced_markets=None):
```
Comment (Herklos) [inline tests/exchanges/__init__.py:54]: 👍

File: tests/exchanges/connectors/ccxt/mock_exchanges_data.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+
+MOCKED_EXCHANGE_INFO = {
```
Comment (Herklos) [inline tests/exchanges/connectors/ccxt/mock_exchanges_data.py:18]: 😅

**Review by Herklos — APPROVED** [review]
👍

File: tests/personal_data/portfolios/test_portfolio.py [hunk 3]
```diff
@@ -64,18 +64,34 @@ async def test_copy_portfolio(backtesting_trader):
 
 
 async def test_get_portfolio_from_amount_dict(backtesting_trader):
+    # todo fix
```
Comment (Herklos) [inline tests/personal_data/portfolios/test_portfolio.py:67]: Is it fixed ?

Comment (GuillaumeDSM) [reply tests/personal_data/portfolios/test_portfolio.py:67]: woops, yes it is

Comment (GuillaumeDSM) [reply tests/personal_data/portfolios/test_portfolio.py:67]: remove here https://github.com/Drakkar-Software/OctoBot-Trading/pull/938

**Pull request merged**